usrp_drivers/n200/n200_driver.cpp

```diff
@@ -659,6 +609,12 @@ int UHD_SAFE_MAIN(int argc, char *argv[]) {
   threads.push_back(std::move(transmit_t));
   threads.push_back(std::move(receive_t));
 
+  auto set_tid_msg = std::string("SET_TIDS");
+  SEND_REQUEST(driver_to_mainaffinity,
+                              driver_options.get_mainaffinity_to_driver_identity(), set_tid_msg);
+  auto keep_going = RECV_REPLY(driver_to_mainaffinity,
```

Comment:
Are you intending to use the keep_going variable?

Reply:
Nope. I removed it.